### ISSUE TITLE
Graph API for Chat not sending notification except for request with validation token.

### ISSUE THREAD

**Issue by damiankucharski** [MicrosoftDocs/msteams-docs#11679]
### Steps to reproduce

We are not getting notifications for new teams messages. The MS Graph subscription (chats/getAllMessages?model=B) is active and we also tried renewing it.
The validation token is received after the subscription is being created but later no other notification is being received.
We did not change anything with our system, the issue appeared today, yesterday everything was working.

### Expected behavior

After we send a message in teams, the notification is received as POST request by our API endpoint as usual.

### Actual behavior

API does not receive any requests besides the single validation token request:
2024-10-10 09:37:49 INFO     got val string request. Source: subscription was requested to made
INFO:     52.142.115.31:0 - "POST /graph/notifications?validationToken=Validation%3a+Testing+client+application+reachability+for+subscription+Request-Id%3a+185e6c58-3c20-4adb-ad67-14c878280072 HTTP/1.1" 200 OK

**Comment by MMarciniak103** [comment]
Hey, I am colleague from Damian's teams.
Additional context: we are also using mailFolders('inbox')/messages subscription and this one works correctly. It is only the `chats/getAllMessages?model=B` that stopped sending notifications

**Comment by Prasad-MSFT** [comment]
Ensure that the subscription is being renewed before it expires and that the renewal requests are being processed successfully.
Also, are you getting any of these errors? https://learn.microsoft.com/en-us/graph/api/chats-getallmessages?view=graph-rest-1.0&tabs=http#errors

**Comment by MMarciniak103** [comment]
Hey,
The subscription is always renewed before it expires (by lifecycle handler passed in "lifecycleNotificationUrl"). 
As for the `https://graph.microsoft.com/v1.0/users/{user_id}/chats/getAllMessages?$top=2` we cannot run it in delegated mode:
{'error': {'code': 'PreconditionFailed',
  'message': 'Requested API is not supported in delegated context',
  'innerError': {'date': '2024-10-10T11:14:35',
   'request-id': '8596cae3-91ca-41d4-bcd9-87dfb3fbc213',
   'client-request-id': '8596cae3-91ca-41d4-bcd9-87dfb3fbc213'}}}
   
 but we can confirm that this account have valid office365 licence (it was working for couple of months now, it just stopped suddenly today)

**Comment by MMarciniak103** [comment]
I've also tried running `https://graph.microsoft.com/v1.0/chats/{chat_id}/messages?$top=2` and it runs successfully

**Comment by MMarciniak103** [comment]
It looks like the similar issue as here: https://github.com/MicrosoftDocs/msteams-docs/issues/11558
although we stopped getting any notifications, while this guys was missing some of them.

**Comment by knmarcin** [comment]
In my project we have same problem. After couple of months, teams channel messages stopped coming.

**Comment by knmarcin** [comment]
I think it stopped working around on Oct 9, 2024, 20:00 CEST. Then barely promile of exchanged messages came through my endpoint.

**Comment by MMarciniak103** [comment]
Hey @knmarcin, it aligns with our observations. Since the beginning of this week (around October 7/8), we began to notice a problem such that some messages were not sent to us, until yesterday we stopped getting any.

**Comment by knmarcin** [comment]
Yes, the last message we processed was yesterday at 10:07 CEST. Since then, no further messages have been received

**Comment by damiankucharski** [comment]
@Prasad-MSFT it looks like this may be a global issue, see this link: https://learn.microsoft.com/en-us/answers/questions/2101043/frequent-interruptions-in-graph-api-subscriptions

**Comment by Prasad-MSFT** [comment]
Hi @damiankucharski , @MMarciniak103 , @knmarcin - we reached out to engineering team regarding this issue. Once we have any update from them, we will share in this thread. Thanks!

**Comment by laknoll** [comment]
Just to clarify, for us it’s not getAllMessages that fails, but /teams/{id}/channels/{id}/messages.

**Comment by Prasad-MSFT** [comment]
**Update:**

We have raised an ICM for this issue and engineering team is looking into it. We will inform you once we have further update. Thanks!

**Comment by damiankucharski** [comment]
Thanks!

**Comment by hanord** [comment]
It seems like we're getting notifications for message subscriptions again. Has the issue been fixed, @Prasad-MSFT ?

**Comment by bistech-rcampbell** [comment]
Still having the same issue over here but for subscriptions to Microsoft Teams transcripts, tested as of this morning and the service is still not working, I imagine it's the same issue as this. Still happening as of this morning.

Same issue as Damian reports

> The validation token is received after the subscription is being created but later no other notification is being received.
We did not change anything with our system.

**Comment by Prasad-MSFT** [comment]
Hi @damiankucharski , @bistech-rcampbell - Could you please share an example subscription/tenant id with a sample successful notification that was received? We'll first need to validate whether or not we are receiving any notifications for the subscription from Teams.

**Comment by bistech-rcampbell** [comment]
Hey @Prasad-MSFT Raised a ticket separately to this and has been resolved as a result, thanks

**Comment by MMarciniak103** [comment]
Hey @Prasad-MSFT,
Example of successful notification for subscription id: 7378065f-39f2-46be-8c35-bd86fecd8910:

`
{'value': [{'subscriptionId': '7378065f-39f2-46be-8c35-bd86fecd8910', 'changeType': 'created', 'clientState': '[REDACTED]', 'subscriptionExpirationDateTime': '2024-10-25T13:11:15-08:00', 'resource': "chats('19:9c5c5fb5-1b41-4566-86c9-b282a9854067_9c8cfb81-5423-416a-8f82-ccc6ed0a14e8@unq.gbl.spaces')/messages('1729760713464')", 'resourceData': {'id': '1729760713464', '@odata.type': '#Microsoft.Graph.chatMessage', '@odata.id': "chats('19:9c5c5fb5-1b41-4566-86c9-b282a9854067_9c8cfb81-5423-416a-8f82-ccc6ed0a14e8@unq.gbl.spaces')/messages('1729760713464')"}, 'encryptedContent': None, 'tenantId': '8f0bf071-57ab-4b48-8504-cd24bb511bee'}]}
`

It looks like it works for some time now. Do you know what was the source of problem?

**Comment by damiankucharski** [comment]
@Prasad-MSFT just bumping up @MMarciniak103's reply as bot marked the thread as stale because it was not me who answered your original question.

**Comment by damiankucharski** [comment]
Hi @Prasad-MSFT, it looks to work correctly now. Do you know what might have been the underlying cause?

**Comment by Prasad-MSFT** [comment]
@damiankucharski - Apologies, but currently there is no such information available from engineering team side.

**Comment by Prasad-MSFT** [comment]
We are closing this issue for now. Please feel free to reopen if required. Thanks!
